### PR TITLE
Full fertilizer bottles of type glass/bottle

### PR DESCRIPTION
**Pull request by Hubblenaut**
- Fertilizer now of type glass/bottle and spawns with 60 units
 - Fertilizer bottle defines moved to chemistry bottle defines
 - Amount available in vendor roughly divided by 6
 - Cost of fertilizer in biogenerator multiplied by 6

**Note:** Apparently the three types of fertilizer have vastly different production costs, but all the same effect on a hydroponics tray. Something to inspect in the future.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -143,6 +143,39 @@
 		reagents.add_reagent("ammonia", 60)
 		update_icon()
 
+/obj/item/weapon/reagent_containers/glass/bottle/eznutrient
+	name = "\improper EZ NUtrient bottle"
+	desc = "A small bottle."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle-4"
+
+	New()
```
Comment (Kelenius) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: No

**Comment by Kelenius** [comment]
>all the same effect on a hydroponics tray

They're different, it's just hidden in very obscure code: https://github.com/PolarisSS13/Polaris/blob/master/code/modules/hydroponics/trays/tray.dm#L115

File: code/modules/reagents/reagent_containers/glass/bottle.dm (hunk 1 above)
Comment (Hubblenaut) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: Adjusting all reagent containers to current code standards is not in scope of this PR.

**Comment by Nerezza** [comment]
There's a big list of chem effects on plants hidden in the Bay wiki: https://wiki.baystation12.net/Guide_to_Hydroponics#Chemicals.2C_Wonderful_Chemicals

**Comment by Kelenius** [comment]
>negligible

REEEEEEEEEEEEEEEEEEE

**Comment by Hubblenaut** [comment]
Updated bottle.dm to use absolute pathings.

**Pull request merged**